### PR TITLE
Add persist configuration for store

### PR DESCRIPTION
**Pull request by loc-nguyenthien**

### CODE REVIEW

**Comment by honghoangsts** [comment]
We need add `REHYDRATE` for reducer.

**Review by honghoangsts — CHANGES_REQUESTED** [review]

**Review by vunguyendhSTS — APPROVED** [review]

**Comment by loido** [comment]
Is it merge into branch master?

**Review by loido — APPROVED** [review]

File: template/src/store/reducers/index.js [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+}
+
+const userPersistConfig = {
+  key: 'user',
+  storage: AsyncStorage,
+}
+
 export default combineReducers({
-  user,
+  user: persistReducer(userPersistConfig, user),
```
[the remark this answers is not in the document]
Comment (baonguyenhsts) [reply template/src/store/reducers/index.js:27]: I think he wants to use Nested Persists
https://github.com/rt2zz/redux-persist#nested-persists

File: template/src/store/store.js [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+  reducer: persistedReducer,
   middleware: getDefaultMiddleware => getDefaultMiddleware({serializableCheck: false, thunk: false}),
   enhancers: [middlewareEnhancer],
 })
 
 sagaMiddleware.run(rootSaga)
 
+export const persistor = persistStore(store)
+
 export {store}
```
[the remark this answers is not in the document]
Comment (vunguyendhSTS) [reply template/src/store/store.js]: seem it changed, please check it in latest commit @honghoangsts

**Review by honghoangsts — APPROVED** [review]

**Pull request merged**